### ISSUE TITLE
Provide a way to copy feed URL

### ISSUE THREAD

**Issue by RalfJung** [shgysk8zer0/awesome-rss#128]
**Is your feature request related to a problem? Please describe.**
I use an external local feedreader, so the thing I really want to do with an RSS feed is copy its URL to the clipboard so that I can paste it into the other application (or my Podcast reader on my phone).  Currently, however, when I click an RSS feed in Awesome RSS's menu, a download window opens and it lets me download the .atom file -- that's not very useful.

**Describe the solution you'd like**
Some way of copying the URL instead of "opening" it would be great. When there is more than one feed, this is actually possible by opening the menu and then right-clicking the item, but when there is just one item there is no menu and hence no way to right-click.
So maybe the menu could just always appear, even if there is just one feed, to provide this option? That is still not very discoverable though.
Or maybe "subscribe using" could offer to open a window showing the URL?

**Describe alternatives you've considered**
See above.

**Additional context**
Currently I go to the site source code and copy-paste the URL from there. Lucky enough this is not something I have to do every day.

**Comment by Park0** [comment]
I think your looking for this: https://addons.mozilla.org/en-US/firefox/addon/rsspreview/
The atom download comes from a change in firefox not have a rsspreview feature anymore. It has nothing todo with awesome-rss.

**Comment by RalfJung** [comment]
> The atom download comes from a change in firefox not have a rsspreview feature anymore. It has nothing todo with awesome-rss.

I am aware. I am basically asking for Awesome RSS to account for this change so that it remains as useful as it used to be. (I didn't really care about the rsspreview before, but it *did* give me a chance to copy the URL from the address bar.)

I'll try the addon you suggested, thanks! But I was not asking for a full-blown preview above, just for a way to obtain the URL.

**Comment by Park0** [comment]
> 
> But I was not asking for a full-blown preview above, just for a way to obtain the URL.

That's the way it was done in the old Firefox. And it will allow u to copy the url. Maybe there is one that will only show the xml file. But the difference will be minimal. 

The only way i can think of making it better is by providing a way to connect to your local rss reader. But that depends on Reader and Operation system if and how this is possible. 

For now i only see web-based RSS readers with integration. I am not sure if it is possible at all for a local client.

**Comment by RalfJung** [comment]
An "integration" option that consists solely of putting the URL into the clipboard (with some notification that this has happened) instead of opening it would already be enough.

**Comment by Park0** [comment]
@RalfJung could u try my pull request?

**Comment by RalfJung** [comment]
How would I do that? Is there an xpi provided somewhere?

I have since then switched entirely to "RSSPreview", which nowadays also has an icon+menu in the address bar.

**Comment by Park0** [comment]
The only way to test it is to run it from source in firefox. I can't make a release that would be excepted by firefox. I can make a ipx/zip but it would not install without setting firefox to developer mode. 

@shgysk8zer0 could u check my PR?

**Comment by selivan** [comment]
This extension seems abandoned by its author, but thanks to this thread I switched to "Want My RSS": https://addons.mozilla.org/en-US/firefox/addon/want-my-rss/

It shows feed preview from which you can copy address. Also supports some popular news readers and allows to define custom subscribe URL.

**Comment by winny-** [comment]
Yup, seems reasonable to assume not all RSS feeds are served with a MIME type that makes sense.  I've seen RSS feeds in the wild served up with application/octect-stream.  The only way to address this problem is to provide a way to grab the URL and do stuff to it.  Any other solution will not account for the murkiness of RSS MIME types.

**Comment by igorlogius** [comment]
@winny- 

>  I've seen RSS feeds in the wild served up with **application/octect-stream**.

Just by chance stumbeld over this thread and over your comment. 
I'd seriously like to know which site/frameworks does this and i kind of hope its "just a misconfiguration". Would you mind sharing a link?

The reason i am interested is because i maintain this [feed detection addon](https://addons.mozilla.org/firefox/addon/find-feeds/) which has the core premise that the content-type would at least be somewhat reliable.  

- How silly of me for thinking that, right?! 🥲
